components/mixins/experience-event/industry-verticals/experienceevent-bill-pay-details.schema.json

```diff
@@ -12,9 +12,13 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:tags": {
-    "industry": ["financial_services"]
+    "industry": [
+      "financial_services"
+    ]
   },
-  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
   "description": "Bill pay steps such as start, save, failure, and complete.",
```

Comment:
We need to update the description on this mixin now that the content has shifted dramatically. Let's put:
Bill pay details such as transaction information, service account ID, and balance.